src/GitLink/Linker.cs

```diff
@@ -131,5 +162,41 @@ public static bool Link(string pdbPath, LinkOptions options = default(LinkOption
 
             return true;
         }
+
+        private static void CreateSrcSrv(string srcsrvFile, SrcSrvContext srcSrvContext)
+        {
+            Argument.IsNotNull(() => srcSrvContext);
+            Argument.IsNotNullOrWhitespace(() => srcSrvContext.RawUrl);
```

Comment:
Replace with Argument.IsNotNullOrWhitespace("srcSrvContext.RawUrl", srcSrvContext.RawUrl); 

Nested expressions are not supported (yet) in Catel.Fody and causes invalid IL.